codespell_lib/data/dictionary.txt

```diff
@@ -17125,7 +17138,9 @@ postcondtions->postconditions
 Postdam->Potsdam
 postgress->postgresql
 postgressql->PostgreSQL
+postgrsql->postgresql
```

Comment:
Capitalise this and the top one to match the middle one.

Reply:
d71d521